client/src/pages/NotFound/NotFound.styles.scss

```diff
@@ -12,286 +12,3 @@ html, body {
 * {
   box-sizing: border-box;
 }
```

Comment:
consider either removing or moving this into the component itself. This should allow us to the scss file